### DISCUSSION TITLE
[0.13.0] Automate Timelapse Export

### DISCUSSION THREAD

**Discussion by weitheng** [blakeblackshear/frigate#8204] [Ask A Question]
I am aware that in v0.13.0, there's a new [export API](https://deploy-preview-6262--frigate-docs.netlify.app/integrations/api/#post-apiexportcamerastartstart-timestampendend-timestamp) to export recordings as a Timelapse. However, I am unsure what the format of `<start-timestamp>` and `<end-timestamp>` should be, and how it would be possible to create an automation in Home Assistant to automate this and send the timelapse to a messaging platform like Telegram, since there's not GET API to download the Timelapse.

**Comment by NickM-27** [comment]
They are both unix timestamps, the latest version of the frigate integration includes a service to generate the timelapse. Currently there is on direct access in the integration to download it

**Comment by vajonam** [comment]
@NickM-27  can the response include the filename, this will allow it to easier used in other export scripts?  also maybe call back when export is complete with t he filename, if one wants to move the file? are any of these in the pipe?

**Reply by NickM-27** [reply]
I mean PRs are always welcome. I am not sure what you mean by callback; probably easier to have an mqtt topic for exports

**Reply by vajonam** [reply]
I was thinking, something like an additional POST param that will include a callback URL or an mqtt topic, that would either send callback to the URL specific in the POST params or post a response include filename created to the mqtt topic. . if the integration include a download mechanism it can provide a full URL as well. I will try and cook something up. Thanks
